common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1296,8 +1296,11 @@
     public static final KnownKey zm_oauth_classes_handlers_yahoo = KnownKey.newKey("com.zimbra.oauth.handlers.impl.YahooOAuth2Handler");
     public static final KnownKey zm_oauth_classes_handlers_google = KnownKey.newKey("com.zimbra.oauth.handlers.impl.GoogleOAuth2Handler");
     public static final KnownKey zm_oauth_classes_handlers_facebook = KnownKey.newKey("com.zimbra.oauth.handlers.impl.FacebookOAuth2Handler");
-    
-    
+
+    // Feed Manager comma-separated ip blacklist
+    public static final KnownKey zimbra_feed_manager_ip_blacklist = KnownKey.newKey("");
```

Comment:
I think we should add "localhost" and "127.0.0.1" as default values. Others can be added later by admin. What do you say?

Reply:
Those are covered by link local.